torch/_dynamo/variables/builder.py

```diff
@@ -564,6 +574,12 @@ def wrap_tensor(self, value: torch.Tensor):
                     guards=self.make_guards(GuardBuilder.TENSOR_MATCH),
                     should_specialize=self.tensor_should_specialize(),
                 )
+            if graph_arg and config.fake_tensor_propagation:
+                fake_tensor_value = tensor_variable.proxy.node.meta["example_value"]
+                if isinstance(
+                    fake_tensor_value, torch._subclasses.fake_tensor.FakeTensor
+                ):
+                    graph_arg.fake_tensor = fake_tensor_value
```

Comment:
@jansel lmk if you think this is the best way to get the fake tensor onto Grapharg

Reply:
Seems fine, might be cleaner to do it closer to when it is constructed.